src/views/TechnicalAssistance/RequestForm/SubmittedRequest.tsx

```diff
@@ -0,0 +1,298 @@
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+import { Grid } from '@trussworks/react-uswds';
+import { camelCase, capitalize } from 'lodash';
+import { DateTime } from 'luxon';
+
+import UswdsReactLink from 'components/LinkWrapper';
+import Divider from 'components/shared/Divider';
+import useTRBAttendees from 'hooks/useTRBAttendees';
+import {
+  GetTrbRequest_trbRequest as TrbRequest,
+  GetTrbRequest_trbRequest_form as TrbRequestForm
+} from 'queries/types/GetTrbRequest';
+import { TRBWhereInProcessOption } from 'types/graphql-global-types';
+import { formatDateUtc } from 'utils/date';
+
+import { AttendeesTable } from './AttendeesForm/components';
+import DocumentsTable from './DocumentsTable';
+
+function SubjectDefinition(
+  form: any,
+  field: keyof TrbRequestForm
+): React.ReactNode {
+  const { t } = useTranslation('technicalAssistance');
+  return Array.isArray(form[field]) && form[field].length ? (
+    form[field]
+      .map((v: string) =>
+        v === 'OTHER'
+          ? `${t('basic.options.other')}: ${form[`${field}Other`]}`
+          : t(`subject.options.${field}.${v}`)
+      )
+      .join(', ')
+  ) : (
+    <em>{t('check.noTopicsSelected')}</em>
+  );
+}
+
+type SubmittedRequestProps = {
+  request: TrbRequest;
+  showEditSectionLinks?: boolean;
+};
+
+function SubmittedRequest({
+  request,
+  showEditSectionLinks
+}: SubmittedRequestProps) {
+  const { t } = useTranslation('technicalAssistance');
+
+  const {
+    data: { attendees }
+  } = useTRBAttendees(request.id);
+
+  return (
+    <>
+      <dl className="easi-dl margin-top-4 margin-bottom-3">
+        <Grid row gap>
+          <Grid tablet={{ col: 12 }} desktop={{ col: 6 }}>
+            <dt>{t('table.header.submissionDate')}</dt>
+            <dd>
+              {request.form.submittedAt
+                ? DateTime.fromISO(request.form.submittedAt).toFormat(
+                    'MMMM d, yyyy'
+                  )
```

Comment:
In other parts you've used the util `formatDateLocal`, and here is a different implementation.  Just a nitpick on standardization

Reply:
Thanks for finding this. I missed it during a merge conflict. Changed it to `formatDateLocal`.